src/goto-symex/html.cpp

```diff
@@ -10,14 +10,33 @@
 #include <optional>
 #include <filesystem>
 #include <util/cmdline.h>
+#include <nlohmann/json.hpp>
+#include <regex>
+#include <array>
+#include <string>
+#include <set>
+#include <vector>
+#include <numeric>
+#include <iostream>
+
+using json = nlohmann::json;
+
+static json tests_json;
+static bool first_write = true;
+static std::map<std::string, std::vector<std::string>> source_files;
```

Comment:
Avoid global (and static) variables. If you really need it, use a static inline inside a class